pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -583,9 +585,43 @@ self: super:
         propagatedBuildInputs = [ libpciaccess epoxy ] ++ commonPropagatedBuildInputs ++ lib.optionals stdenv.isLinux [
           udev
         ];
-        prePatch = stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
+        prePatch = (attrs.prePatch or "") + stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
           export CFLAGS+=" -D__uid_t=uid_t -D__gid_t=gid_t"
         '';
+        patches = [
+          # NVIDIA PRIME Render Offload patchset
+          # Should be upstream by 1.21
+          # http://download.nvidia.com/XFree86/Linux-x86_64/435.17/README/primerenderoffload.html
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/7f962c70b6d9c346477f23f6c15211e749110078.patch";
+            sha256 = "196nzhw4wipz6a57j1k6wrp4j80z1fq4fpqhw3xcc1mb3yy24pf6";
+          })
+
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/37a36a6b5b887d5c5a17a6931ceba8ad5d1bb6d5.patch";
+            sha256 = "1q518hwzi7ry4xzydvsghnyb3iz175iwbzqqml1q2s2v5mrc0qjz";
+          })
+
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/8b67ec7cc6fda243480a5a8ca118b66242f3eb2c.patch";
+            sha256 = "1l29x87vppfwkr2lhijwvypp1ccsfgd9kwx4dk0mskdalspfc2bd";
+          })
+
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/56c0a71fdd94a008e5d746261f70a713c4767f93.patch";
+            sha256 = "177cgpz4w5b46p5pa81p9fnyzqvsb4ji7yfnbkb8n3clw3mpp9iz";
+          })
+
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/b4231d69028adc8123801a7552b40a15ea928d1b.patch";
+            sha256 = "1qxwjs007jg4ahcvi0s52sq869l07ycd0qkjx1ckspiy1ab28j1m";
+          })
+
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/7c3f316d07753e3b3a6d23e629ce8e944820c225.patch";
+            sha256 = "1p17p00xlc9wshr7xqqz2064p88axfrjwfld68hakrzivil4awbp";
```

Comment:
hash mismatch in fixed-output derivation '/nix/store/vi1hjpz012fr6risfc1cnqfhw4ryks5d-7c3f316d07753e3b3a6d23e629ce8e944820c225.patch':
  wanted: sha256:1p17p00xlc9wshr7xqqz2064p88axfrjwfld68hakrzivil4awbp
  got:    sha256:19199ynzxhx13m6rb0ni9d4rrv19d5ymvlspd35dh0yidhmipafz

Reply:
What is this patch? Its not mentioned in http://us.download.nvidia.com/XFree86/Linux-x86_64/435.21/README/primerenderoffload.html